common/federation/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file? We don't seem to have logs let alone tests in this module.